### PR TITLE
docs: Clarifying analytics detail

### LINKED ISSUE #1286: Device data collected by the supervisor: Should remove the "(if allowed)" part
In the Collected Data page we mention:
> Data is submitted to Mixpanel from balena-managed devices (if allowed) on the events listed in the table below.

Since we do not offer a way to users to opt out their devices from sending analytics, we should remove the `(if allowed)` part from that sentence.

See: https://www.balena.io/docs/learn/more/collected-data/

### PR DESCRIPTION
**Pull request by garethtdavies**
Closes #1286 

Change-type: patch
Connects-to: #1286 
Signed-off-by: Gareth Davies gareth@balena.io

### CODE REVIEW

**Comment by garethtdavies** [comment]
@balena-ci retest

File: pages/learn/more/collected-data.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ collected too.
 
 ## Device data collected by the supervisor
 
-Data is submitted to Mixpanel from {{ $names.company.lower }}-managed devices (if allowed) on the events listed in
+Data is submitted to Mixpanel from {{ $names.company.lower }}-managed devices on the events listed in
```
Comment (xginn8) [inline pages/learn/more/collected-data.md]:
```suggestion
Data are submitted to Mixpanel from {{ $names.company.lower }}-managed devices on the events listed in
```

**Comment by garethtdavies** [comment]
@balena-ci retest

**Comment by lurch** [comment]
I've just noticed that in the section about Etcher it says "In addition to the data described above..." - however AFAIK the usage data collected by Etcher is anonymised, and therefore **doesn't** include usernames, email addresses or company names? And I think the data transmitted by Etcher _might_ also include the filename of the image being flashed?
ping @thundron and @zvin

**Pull request merged**